Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -3987,34 +3980,9 @@ void HTMLMediaElement::setSeeking(bool seeking)
     m_seeking = seeking;
 }
 
-void HTMLMediaElement::refreshCachedTime() const
-{
-    if (!m_player)
-        return;
-
-    m_cachedTime = m_player->currentTime();
-    if (!m_cachedTime) {
-        // Do not use m_cachedTime until the media engine returns a non-zero value because we can't
-        // estimate current time until playback actually begins.
-        invalidateCachedTime();
-        return;
-    }
-
-    m_clockTimeAtLastCachedTimeUpdate = MonotonicTime::now();
-}
-
 void HTMLMediaElement::invalidateCachedTime() const
```

Comment:
As we discussed, I think you should remove the _all_ of the caching code from HTMLMediaElement